src/main/resources/db/migration/2004019.sql

```diff
@@ -0,0 +1,3 @@
+# 2.4.19
```

Comment:
@alexander-kuruvilla The latest version now is 2.4.20, so you can rename the file to `2004020.sql`.